lmdeploy/cli/serve.py

```diff
@@ -349,28 +350,56 @@ def api_server(args):
 
         from lmdeploy.messages import VisionConfig
         vision_config = VisionConfig(args.vision_max_batch_size)
-        run_api_server(args.model_path,
-                       model_name=args.model_name,
-                       backend=backend,
-                       backend_config=backend_config,
-                       chat_template_config=chat_template_config,
-                       vision_config=vision_config,
-                       server_name=args.server_name,
-                       server_port=args.server_port,
-                       allow_origins=args.allow_origins,
-                       allow_credentials=args.allow_credentials,
-                       allow_methods=args.allow_methods,
-                       allow_headers=args.allow_headers,
-                       allow_terminate_by_client=args.allow_terminate_by_client,
```

Comment:
Is “allow_terminate_by_client” intentionally deleted?

Reply:
mistakes ...